config/apprunner/apprunner-default-config.yaml

```diff
@@ -0,0 +1,35 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:55681
```

Comment:
If this feature hasn't been released yet, is there any way we can use the official 4318 port for this instead of this legacy one?

Reply:
do you know if OTel SDK has updated its default HTTP port to `4318` yet? I previously only updated OTel SDK for gRPC on new port. Thanks